forge-game/src/main/java/forge/game/ability/effects/MakeCardEffect.java

```diff
@@ -95,6 +95,9 @@ public void resolve(SpellAbility sa) {
                         game.getAction().moveTo(ZoneType.None, card, sa, moveParams);
                         cards.add(card);
                         toMake--;
+	                        if (sa.hasParam("Tapped")) {
```

Comment:
the code formatting for this is a bit off, @tool4ever or another might fix that later with another MR